config/default.yml

```
@@ -38,6 +38,16 @@ Lint/NumberConversion:
     - fortnights
     - in_milliseconds
 
+Rails/ActionControllerTesting:
+  Description: 'Use `ActionDispatch::IntegrationTest` instead of `ActionController::TestCase`.'
+  Reference: 'https://api.rubyonrails.org/classes/ActionController/TestCase.html'
+  StyleGuide: 'https://rails.rubystyle.guide/#integration-testing'
```

Can you order to the following?

```suggestion
  Description: 'Use `ActionDispatch::IntegrationTest` instead of `ActionController::TestCase`.'
  StyleGuide: 'https://rails.rubystyle.guide/#integration-testing'
  Reference: 'https://api.rubyonrails.org/classes/ActionController/TestCase.html'
```